src/environments/crud.urls.js

```diff
@@ -1,19 +1,25 @@
-import Method from "axios";
-
-export const crudUrls = {
```

Comment:
genre and borrower use baseport, basehost, prefix -  these changes cause genre and borrowers to not load